Assets/Prefabs/Cutscenes/Cutscene Marshmallow.prefab

```diff
@@ -166,6 +165,7 @@ MonoBehaviour:
   - {fileID: 8300000, guid: e9de9b181cfc6804e9e615db52c9d7ad, type: 3}
   hangOffset: {x: 0, y: 0}
   swingDeltaMultiplier: 0.5
+  speedRingSpeedThreshold: 100000
```

Comment:
cutscene marshmallow should be a prefab variant but oh well we doing this instead